docs/sources-asciidoc/src/main/asciidoc/concept-chapter-Configuring_the_Media_Server.adoc

```diff
@@ -279,3 +279,34 @@ The more resources you have pre-configured on startup in the resource pool, the
 It is up to the user to decide the best trade-off for the setup (greater memory usage on startup vs slower response when new resources are required in runtime).
 ====
 
+== DTLS Configuration
+
+The DTLS configuration allows to define some properties that are used by WebRTC resources, through the DTLS protocol. Since the nature of this configuration is strongly related with security principles, *some of the default values of this configuration need to be replaced by the user's private information, ensuring that no vulnerabilities could be exploited by someone else.* This refers mainly to the *certificate*, that is shipped with mediaserver for demonstration purposes only, and its better explained in the *Certificate* subsection.
+
+.DTLS default configuration
+====
+[source,xml]
+----
+<dtls>
+	<minVersion>1.0</minVersion>
+	<maxVersion>1.2</maxVersion>
+	<cipherSuites>TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256, 
+		TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA, TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA</cipherSuites>
+	<certificate path="../conf/dtls/x509-server-ecdsa.pem" key="../conf/dtls/x509-server-key-ecdsa.pem" algorithm="ecdsa"/>
+</dtls>
+----
+====
+
+=== DTLS Version
+
+The properties *minVersion* and *maxVersion* define the supported minimum and maximum version of the DTLS protocol, respectively. The possible values suported by each property is *1.0* or *1.2*. 
+
+=== Cipher suites
+
+The *cipherSuites* property allows to inform one or more cipher suites to be used by DTLS. The names of the suites follows the JSSE cipher suite naming standard (see https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#ciphersuites) and must be separated by comma.
+
+=== Certificate
+
+This property allows to set the absolute or relative path of the certificate and key files. The path for the certificate file must be informed in the *path* property, and the path for the key file must be informed in the *key* property. The signature algorithm used for the certificates must be informed at the *algorithm* property, which supports the values *rsa* and *ecdsa*. The default certificate files must be replaced by certificates generated by the user, as described by the following warning:
```

Comment:
Present each certificate, explaining one is for ECDSA and the other for RSA.
At end of chapter, give examples on how to setup DTLS for ECDSA and RSA, respectively.